src/components/ListWraper/ListWraper.js

```diff
@@ -1,33 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ListItem from './ListItem/ListItem';
 
-const ListWrapper = ({ items }) => {
-  const { header, users } = items;
+class ListWrapper extends Component {
+ 
+  render(){
+    const { header, users } = this.props.items;
```

Comment:
const { items: { header, users } } = this.props;

Reply:
zrobione